### PR TITLE
NEPT-1059: Add the breadcrumb block into the proper europa region.

### PR DESCRIPTION
**Pull request by Fefaine**
### Description

Add the breadcrumb block inside the header bottom region

### Change log

- Added: hook_themes_enabled

### Commands

```sh
no command to run

```

### CODE REVIEW

File: profiles/common/modules/features/cce_basic_config/cce_basic_config.module [hunk 1]
```diff
@@ -358,3 +358,39 @@ function cce_basic_config_preprocess_container(&$variables) {
     $variables['theme_hook_suggestions'][] = $prefix . $suffix;
   }
 }
+
+/**
+ * Implements hook_themes_enabled().
+ */
+function cce_basic_config_themes_enabled($theme_list) {
+  foreach ($theme_list as $key => $themename) {
```
Comment (drupol) [inline profiles/common/modules/features/cce_basic_config/cce_basic_config.module]: If `$key` is not used, then you could also remove it from the `foreach`.

File: profiles/common/modules/features/cce_basic_config/cce_basic_config.module [hunk 2]
```diff
@@ -358,3 +358,39 @@ function cce_basic_config_preprocess_container(&$variables) {
...
+}
+
+/**
+ * Custom function that moves the proper blocks in the right areas.
+ */
+function _cce_basic_config_europa_breadcrumbs() {
+  // Add block in site-wide context.
+  $context = module_invoke('context', 'load', 'site_wide');
+
+  $block_to_add = array(
```
Comment (drupol) [inline profiles/common/modules/features/cce_basic_config/cce_basic_config.module]: There is no need to have a variable for this.

**Comment by voidtek** [comment]
The branch need to be rebased.

**Comment by drupol** [comment]
I'm doing it.

File: profiles/common/modules/features/cce_basic_config/cce_basic_config.module [hunk 3]
```diff
@@ -341,3 +341,37 @@ function cce_basic_config_modules_enabled($modules) {
...
+  _cce_basic_config_europa_breadcrumbs();
+}
+
+/**
+ * Custom function that moves the proper blocks in the right areas.
+ */
+function _cce_basic_config_europa_breadcrumbs() {
+  // Add block in site-wide context.
+  $context = module_invoke('context', 'load', 'site_wide');
+  $context->reactions['block']['blocks']['easy_breadcrumb'] = [
```
Comment (drupol) [inline profiles/common/modules/features/cce_basic_config/cce_basic_config.module]: Let's use the regular syntax for arrays :-)

**Comment by Fefaine** [comment]
Ready now

**Comment by drupol** [comment]
👍

**Comment by drupol** [comment]
👍

**Pull request merged**